scripts/build.js

```diff
@@ -7,14 +7,18 @@ const glob = process.argv[2];
 
 process.env.PATH = `./node_modules/.bin:${process.env.PATH}`;
 
-const run = (cmd, async = false) => {
+const run = (cmd) => {
   console.log(`>> ${cmd}`);
   const [p, ...args] = cmd.split(' ');
-  const runner = async ? spawn : spawnSync;
-  runner(p, args, { stdio: 'inherit' });
+  const runner = spawnSync;
+  const { status } = runner(p, args, { stdio: 'inherit' });
+  if (status !== 0) {
+    process.exit(status);
+  }
 };
 
 if (glob) {
+  run(`eslint --color --quiet packages/${glob}/{src,test}/**/*.{js,jsx,ts,tsx}`);
```

Comment:
`nimbus eslint --color ...`

`nimbus` will create eslint config before executing. Otherwise there may not be eslint config as we don't check them in and let nimbus handle that.

Reply:
I wanted to limit to just the specified packages, but `nimbus` doesn't seem to support that for `eslint` command.

Reply:
My bad. It turns out I can just pass the file directory. Previous tried the same `--workspaces=` approach as other `nimbus` command.